template/6-non_functional_requirements.md

```diff
@@ -4,7 +4,28 @@
 
 *Which are the applicable laws and regulations?*
 
-- Security : ensure no user credentials are saved in local or publically displayed
+Our applicatin will collect some personal data such as :
+    - First and Last name (if the user don't use a nickname)
+    - email adresse (necessary for the signIn)
+    - pictures 
+    - conversations (private or group conversation)
+    - locations
+    - affilated university (depending on the group name)
+    - friend relations (we could link the user to his friends/contact)
+    - credential informations (if the user subscrite to some paid functionalities)
+Other personal information could also be entered in the application via the messages, or othe features (to do list, calendar,...). 
+
+For **security** purpose, we should assure that none of these informations are leacked or publically availble. Let's remind that the swiss Federal Act on Data Proectection (FADP) ensure the protection of ["the personality and fundamental rights of natural persons whose personal data is processed"](https://www.google.com/search?client=opera-gx&q=FADP&sourceid=opera&ie=UTF-8&oe=UTF-8). To be more precise, the art.8 of this laws states that *"The controller and the processor shall guarantee a level of data security appropriate to the risk by taking suitable technical and organisational measures"*. If we are still developping application in Switerzland we must apply the mentionned law and assure a proper data protection.
```

Comment:
instead of google link, you should put https://www.fedlex.admin.ch/eli/cc/2022/491/en (a link directly to the law page)